zproject/urls.py

```diff
@@ -547,6 +550,9 @@
     url(r'^calls/zoom/complete$', zerver.views.video_calls.complete_zoom_user),
     url(r'^calls/zoom/deauthorize$', zerver.views.video_calls.deauthorize_zoom_user),
 
+    # Used to join a Big Blue Button video call
+    url(r'^calls/bbb/join$', zerver.views.video_calls.join_bbb),
```

Comment:
Should this be `bigbluebutton`?  We generally avoid abbreviations in our code, so unless `bbb` is how they refer to themselves consistently in code contexts, I'd prefer to expand the name.

Reply:
I think bbb is quite common abbreviations.
Mostly for domains (e.g. bbb.example.com).
The Install Script ist also called (https://github.com/bigbluebutton/bbb-install)[bbb-install].
And most folders in their [main repo](https://github.com/bigbluebutton/bigbluebutton) are called bbb-something.

But if Zulip tries to avoid abbreviations I will change that

Reply:
Yeah, both are probably justifiable; I think it's unlikely we'll much regret either choice.  But if you google for `bbb` or `bbb calls`, you don't find bigbluebutton immediately, so I think the full name has benefits in being more obviously discernable to someone not familiar with it.